### PR TITLE
Fixed:

### PR DESCRIPTION
**Pull request by arbreurkes**
- Show only selected strains in the graph now works when selecting all genomes
- Searching for non-existent genomes in tree now ignores instead of null pointers

### CODE REVIEW

File: src/main/java/application/controllers/TreeController.java [hunk 1]
```diff
@@ -480,35 +480,37 @@ public void selectAll() {
...
-        builder.append("Pyrazinamide: ").append(genome.getPyrazinamide()).append("\n");
-        builder.append("Ofloxacin: ").append(genome.getOfloxacin()).append("\n");
-        builder.append("Rifampin: ").append(genome.getRifampin()).append("\n");
-        builder.append("Streptomycin: ").append(genome.getStreptomycin()).append("\n");
-        builder.append("Spoligotype: ").append(genome.getSpoligotype()).append("\n");
-        builder.append("Genotypic DST: ").append(genome.getGenoDST()).append("\n");
-        builder.append("Tugela Ferry: ").append(genome.isTf()).append("\n");
-
-        mainController.getListFactory().modifyNodeInfo(builder.toString());
+        if(cell.getName().contains("TKK")) {
```
Comment (ghost) [inline src/main/java/application/controllers/TreeController.java]: if and ( should be seperated by a space

**Comment by arbreurkes** [comment]
This PR will become redundant with upcoming changes.

**Comment by arbreurkes** [comment]
Fixes are will most probably not conflict on upcoming rendering update.

**Comment by tieswesterborg** [comment]
Good to me!

**Pull request merged**